OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -19,6 +19,9 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Actor will follow units until in range to attack them.")]
 	public class AttackFollowInfo : AttackBaseInfo
 	{
+		[Desc("Rescanning detects changes in the targets Targetable traits but has a performance hit.")]
+		public readonly bool RescanWhileNotMoving = false;
```

Comment:
`RescanWhileMoving` would be better i think.

Reply:
`RescanWhileMoving` would require a rework of `MoveWithinRange` and have a way bigger performance hit.

Reply:
I mean renaming `RescanWhileNotMoving` to `RescanWhileMoving` and inverting the check and the initial value.

Reply:
There is a big difference between not rescanning at all (=current behaviour), rescanning only while not moving (= PR behaviour enabled) and rescanning only while moving (=your proposal enabled).

Anyways, I will rework the PR in the near future which should make rescanning independent from moving.

Reply:
Ahh, thanks ;).